seedfarmer/cli_groups/_bootstrap_group.py

```diff
@@ -117,10 +128,14 @@ def bootstrap_toolchain(
     if project is None:
         project = _load_project()
     _logger.debug("Bootstrapping a Toolchain account for Project %s", project)
+    if len(policy_arn) > 0 and not as_target:  # type: ignore
+        raise click.ClickException("Cannot set PolicyARNS and not set the -as-target flag")
```

Comment:
rewrite: Cannot set PolicyARNS and when `-as-target` flag is not set?

Reply:
sure